source/extensions/stat_sinks/wasm/config.h

```diff
@@ -13,6 +13,9 @@ namespace Extensions {
 namespace StatSinks {
 namespace Wasm {
 
+// WebAssembly sink
+extern constexpr char Wasm[] = "envoy.stat_sinks.wasm";
```

Comment:
I don't think the name "Wasm" is too broad in this context, plus why extern?

Reply:
@mathetake is right: we don't need "extern" (and that is not in the link I sent you on how to use constexpr for this case).

Reply:
All right, I will remove it.